server/bin/docker/standalone_test.json

```diff
@@ -173,7 +173,7 @@
               "downsampleAggregator": "max",
               "downsampleFillPolicy": "none",
               "downsampleInterval": "1s",
-              "metric": "sys.cpu0.user",
+              "metric": "sys.cpu.user",
```

Comment:
Removed the file.  I already made the changes to grafana/timely-app/src/datasource/dashboards/standalone_test.json